x-pack/plugins/lens/public/editor_frame_service/editor_frame/config_panel/use_focus_update.tsx

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { useEffect, useState, useCallback } from 'react';
+
+const getFirstFocusable = (el: HTMLElement | null) => {
+  const focusableSelector = 'button, [href], input, select, textarea, [tabindex]';
```

Comment:
nit: It's not used in Kibana at the moment, but there's a dependency to `tabbable` already which is also used by EUI for this task. Maybe we can use it here as well.